builder_data.yaml

```
@@ -55,12 +55,13 @@ apt_repo_command: |
     eval $(/usr/bin/keychain -q --agents gpg --eval __GPG_KEY__);
     export GPG_TTY=$(tty);
     if printf -- '%s' "__APT_PLATFORMS__" | egrep -q -- "APT_PLATFORMS"; then
-      sudo -E freight-cache -c /etc/freight.conf.d/community.conf;
+      sudo -E freight-cache -c /etc/freight.conf.d/community.conf &
```

Possibly a mere formality, nevertheless this allows the command to avoid receiving SIGHUP if the controlling tty goes away.

Same for the remaining backgrounded commands.

```suggestion
      nohup sudo -E freight-cache -c /etc/freight.conf.d/community.conf &
```

---

I thought about this and I think I prefer not nohupping so it's easier to tell if something has gone wrong / we're encountering errors / the process has frozen via output in the jenkins job